base/docs/helpdb/Base.jl

```diff
@@ -8719,29 +8719,14 @@ alias the input `x` to modify it in-place.
 filt!
 
 """
-    ascii(::Array{UInt8,1})
+    ascii(s::AbstractString)
```

Comment:
By inline I mean moving the markdown docstring to the source, so we can gradually get rid of this Base.jl file.